src/main/java/com/splunk/kafka/connect/SplunkSinkConnectorConfig.java

```diff
@@ -239,6 +251,9 @@
         headerSource = getString(HEADER_SOURCE_CONF);
         headerSourcetype = getString(HEADER_SOURCETYPE_CONF);
         headerHost = getString(HEADER_HOST_CONF);
+        kerberosPrincipal = getString(KERBEROS_PRINCIPAL);
+        kerberosUser = getString(KERBEROS_USER);
+        kerberosKeytabLocation = getString(KERBEROS_KEYTAB_LOCATION);
```

Comment:
Add validation for kerberos.

Reply:
Can you suggest any reference for validation?